src/main_app/admin/routes/users.py

```
@@ -21,84 +21,70 @@
-        def update_can_run_bg_jobs(user_id: int) -> ResponseReturnValue:
-            desired = 1 if request.form.get("can_run_bg_jobs", "0") == "1" else 0
-            return _update_can_run_bg_jobs(user_id, desired)
+        self.bp.route("/", methods=["GET"])(admin_required(self.dashboard))
+        self.bp.route("/<int:user_id>/can_run_jobs", methods=["POST"])(admin_required(self.update_can_run_jobs))
+        self.bp.route("/<int:user_id>/can_run_bg_jobs", methods=["POST"])(admin_required(self.update_can_run_bg_jobs))
+
+    def dashboard(self) -> str:
+        """Render the user management dashboard."""
+        try:
+            users = self.user_service.list_users()
+        except Exception as e:  # pragma: no cover - defensive guard
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)